backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -424,6 +430,18 @@ public static ValueOrError getValue(Map<String, String> row, String name, boolea
     return new ValueOrError(value, name, isRequired);
   }
 
+  public static ValueOrError getValue(
```

Comment:
for consistency, I'm wondering if we should coerce to lowercase at this point rather than modifying it when we get the value?: https://github.com/CDCgov/prime-simplereport/pull/6778/files#diff-8b15d5be57eafd4b4506c12a4916eec9e80d13a212fb6b2f1032802f6b72e8f0R133 like how we are doing for race, sex etc...